snuba/cli/replacer.py

```diff
@@ -76,7 +76,8 @@ def replacer(
 ) -> None:
 
     from streaming_kafka_consumer import Topic
-    from streaming_kafka_consumer.backends.kafka import KafkaConsumer, TransportError
+    from streaming_kafka_consumer.backends.kafka import KafkaConsumer
+    from streaming_kafka_consumer.errors import TransportError
```

Comment:
IT seems TransportError should be Kafka specific.
So since this library has a streaming component that is abstract and independent from Kafka and a Kafka backend,  why don't we keep the Kafka related errors in the Kafka modules (or maybe in an errors submodule of the Kafka module) ?

Reply:
It is true that TransportError is only ever raised by the Kafka backend right now. But I think I interpreted it as a potentially generic error type that can be used for any kind of transport or connection failure not necessarily specific to Kafka. The other consumer errors `EndOfPartition` and `OffsetOutOfRange` also initially sounded pretty Kafka specific to me, but they are indeed being used in the local backend as well.

Reply:
Makes sense. As long as, when we will have specific Kafka errors, we will move them to the kafka backend module.